client/src/document/molecules/localized-content-note/index.scss

```diff
@@ -1,10 +0,0 @@
-@use "sass:math";
-
-.localized-content-note {
-  border-left: 0;
-
-  &.inline {
-    margin: 0;
-    padding: 0.5rem 1rem;
```

Comment:
It seems no need all CSS styles in this file.

Reply:
What makes you think so?